app/assets/javascripts/components/course_creator/course_dates.jsx

```diff
@@ -1,12 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import DatePicker from '../common/date_picker.jsx';
 import CourseUtils from '../../utils/course_utils.js';
 import CourseDateUtils from '../../utils/course_date_utils.js';
 
 const CourseDates = (props) => {
+  const [hasUpDatedDateProps, setHasUpDatedDateProps] = useState(false);
```

Comment:
I think these should use `Updated` rather than `UpDated`.

Reply:
Okay